README.md

```diff
@@ -43,7 +43,7 @@ for more detailed information about the library.
 
 ### Submitting a Bug Report
 
-Swift Markdown tracks all bug reports with [Swift JIRA](https://bugs.swift.org/).
+Swift Markdown tracks all bug reports with [Swift JIRA](https://github.com/apple/swift-markdown/issues).
```

Comment:
The text of the link should also be changed accordingly. "JIRA" is used for "bugs.swift.org" website.

Also, we should update the link and text in "Submitting a Feature Request" too.

You can refer to the PR below for help.

https://github.com/apple/swift-docc/commit/f60baf619e620198d37839ca59c89697ad29a113#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

Reply:
Made the changes please review.